app/scripts/lib/router.js

```diff
@@ -155,6 +159,13 @@ define(function (require, exports, module) {
       return Backbone.Router.prototype.navigate.call(this, url, options);
     },
 
+    navigateViaServer (url) {
```

Comment:
I realize many of the methods in this file lack jsdocs, can you add one? I'm trying to add them as I go in other PRs too.

And what do you think [about the comment made here about changing the name to navigateToUrl](https://github.com/mozilla/fxa-content-server/pull/4676/files#r98183818)?